### ISSUE TITLE
[PRE REVIEW]: easyDifferentialGeneCoexpression, a handy bioinformatics tool to easily perform differential gene coexpression

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#4165]
**Submitting author:** @davidechicco (<a href="http://orcid.org/0000-0001-9655-7142">Davide Chicco</a>)
**Repository:** <a href="https://github.com/davidechicco/easyDifferentialGeneCoexpression" target ="_blank">https://github.com/davidechicco/easyDifferentialGeneCoexpression</a>
**Version:** v1.0.0
**Editor:** Pending
**Reviewer:** Pending
**Managing EiC:** Daniel S. Katz

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/de67df4d0d55a1d39ef099966317bd4c/status.svg)](https://joss.theoj.org/papers/de67df4d0d55a1d39ef099966317bd4c)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/de67df4d0d55a1d39ef099966317bd4c"><img src="https://joss.theoj.org/papers/de67df4d0d55a1d39ef099966317bd4c/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/de67df4d0d55a1d39ef099966317bd4c/status.svg)](https://joss.theoj.org/papers/de67df4d0d55a1d39ef099966317bd4c)
```

**Author instructions**

Thanks for submitting your paper to JOSS @davidechicco. **Currently, there isn't an JOSS editor assigned** to your paper.

@davidechicco if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission (please start at the bottom of the list).

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
It looks like this paper includes an ORCID (0000-0003-0872-7098) that is probably not correct.

**Comment by whedon** [comment]
Wordcount for `paper.md` is 1031

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/978-1-0716-1839-4_12 is OK
- 10.18129/B9.bioc.annotate is OK
- 10.1093/bioinformatics/btab852 is OK
- 10.1038/nmeth.3252 is OK
- 10.1186/1471-2105-11-183 is OK
- 10.1186/s13059-019-1851-8 is OK
- 10.1186/s12859-015-0582-4 is OK
- 10.1109/TCBB.2019.2893170 is OK
- 10.1186/1471-2105-15-S15-S3 is OK
- 10.1038/s41598-017-05298-w is OK
- 10.1093/bioinformatics/bti722 is OK
- 10.1371/journal.pmed.0020124 is OK
- 10.1371/journal.pbio.1002106 is OK
- 10.1038/s41562-017-0189-z is OK
- 10.1371/journal.pone.0079729 is OK
- 10.18129/B9.bioc.diffcoexp is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.88  T=0.02 s (549.7 files/s, 56621.2 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
R                                5            135             82            276
TeX                              1             21              0            169
Markdown                         2             55              0            139
Bourne Shell                     1             11             10             29
-------------------------------------------------------------------------------
SUM:                             9            222             92            613
-------------------------------------------------------------------------------


Statistical information for the repository '0b32d324ffb542af575042d4' was
gathered on 2022/02/14.
No commited files with the specified extensions were found.
```

**Comment by whedon** [comment]
It looks like this paper includes an ORCID (0000-0003-0872-7098) that is probably not correct.

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.04165/joss.04165/10.21105.joss.04165.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.04165/joss.04165/10.21105.joss.04165.pdf) :page_facing_up: :point_left:

**Comment by danielskatz** [comment]
👋 @davidechicco - This submission does not meet the [substantial scholarly effort](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort) criterion for review by JOSS.  

> Total lines of code (LOC). Submissions under 1000 LOC will usually be flagged, those under 300 LOC will be desk rejected.

Please see https://joss.readthedocs.io/en/latest/submitting.html#other-venues-for-reviewing-and-publishing-software-packages for other suggestions for how you might receive credit for your work.

**Comment by danielskatz** [comment]
@whedon reject

**Comment by whedon** [comment]
Paper rejected.

**Comment by davidechicco** [comment]
Thanks @danielskatz for having considered our article submission. What is JOSS policy for resubmissions? Can we improve our software package and our article and then resubmit it to JOSS in the future, or is it rejected for good?

**Comment by danielskatz** [comment]
If you feel that changes have allowed it to meet the submission criteria, resubmission is fine.

**Comment by davidechicco** [comment]
Thanks @danielskatz for your reply, and sorry for not having noticed the unmet criteria of my package.

I developed my package in R and my collaborator @AbbasAlameer developed a Perl wrapper for it. I counted all my R lines of code of my R files and all his Perl lines of code, and noticed the final sum was around 500, so I thought we were over the 300 code line limit. Now I understand that the code line count refers only to the code files within my GitHub repository. Correct?
